public/app/actions/actions.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
This file can be merged with `index`.
I think there is no sense of using separate index.js if there is only 1 file inside the package